wallet.go

```diff
@@ -584,6 +618,9 @@ func (w *Wallet[C]) Deploy(ctx context.Context) (MetaTxnID, *types.Transaction,
 		RevertOnError: true,
 		To:            walletFactoryAddress,
 		Data:          deploymentData,
+		// TODO: Move this hardcoded gas limit to a configuration
+		// or fix it with a contract patch
+		GasLimit: big.NewInt(131072),
```

Comment:
What do you suggest to do instead? I just copied it from the other deploy function in go-sequence.

Reply:
I need to keep it. It's necessary for unit tests. Otherwise v1 wallets fail to deploy.

Wouldn't increase of this gaslimit to the value that is enough for Arbitrum would solve the problem?